procedures/mobile_automation_setup.rst

```diff
@@ -32,100 +26,62 @@ administrating them in the live Taskcluster website.
-       notifications from
-    8. Use the template from
-       ```fenix-nightly`` <https://taskcluster-web.netlify.com/hooks/project-mobile/fenix-nightly>`__
-       with some modifications:
-
-       1. Use the correct ``payload.image``
-       2. Update the ``payload.command`` to clone and run the correct
-          repository
-       3. Update the ``metadata``
-       4. ``scopes`` should be ``assume:hook-id:project-mobile/$hookId``
-          (using the ``HookId`` you defined above)
-
-6.  Do the same thing, but for non-staging nightlies (e.g.:
-    ```fenix-nightly`` <https://taskcluster-web.netlify.com/hooks/project-mobile/fenix-nightly>`__):
-
-    1. ``HookId`` should be ``$product-nightly``
-    2. ``Name`` should not end with "(staging)"
-    3. Copy the task template from the staging hook, with some
-       modifications
-
-       1. Remove the ``--staging`` parameter from ``payload.command``
-       2. Update ``metadata`` to remove anything mentioning "staging"
-       3. Update the scope to assume from the non-staging hook's role
-
-7.  Create the role for the staging hook (e.g.:
-    ```fenix-nightly-staging`` <https://taskcluster-web.netlify.com/auth/roles/hook-id%3Aproject-mobile%2Ffenix-nightly-staging>`__)
-
-    1. Go to `Roles <https://taskcluster-web.netlify.com/auth/roles>`__
-    2. ``RoleId`` should be idential to your staging hook's ``HookId``,
-       but prefixed with ``hook-id:`` (e.g.:
-       ``hook-id:project-mobile/fenix-nightly-staging``)
-    3. ``Description`` should be something like "Hook for building,
-       signing and uploading (staging) Nightly versions of $product"
-    4. Add all the scopes you'll need to fulfill the requirements in you
-       ``.taskcluster.yml`` file (when ``is_mozilla_mobile_repo`` is
-       ``false``)
-
-8.  Do the same thing, but for non-staging nightlies (e.g.:
-    ```fenix-nightly`` <https://taskcluster-web.netlify.com/auth/roles/hook-id%3Aproject-mobile%2Ffenix-nightly>`__)
-
-    1. Remove mentions of "staging" from each field
-    2. Make sure that all scopes are provided to fulfill requirements in
-       ``.taskcluster.yml`` (when ``is_mozilla_mobile_repo`` is
-       ``true``)
+3.  Add ``.taskcluster.yml`` in the root of the repository. This file
+    tells Taskcluster what to do upon github events happening (push,
+    pr, release, etc). Since we're going to want to run ``taskgraph``
+    to decide what tasks to run, we can take a ``.taskcluster.yml`` from
+    a similarly-configured repository, like `fenix
+    example <https://github.com/mozilla-mobile/fenix/blob/master/.taskcluster.yml>`__)
+
+    -  Update repository and treeherder references to refer to your project,
+       rather than ``fenix``.
+
+4.  Implement ``taskgraph`` configuration for the repository. See the
+    `Fenix configuration <https://github.com/mozilla-mobile/fenix/tree/master/taskcluster>`__.
+    You'll need to implement the following parts:
```

Comment:
Followup for me: Link to the blog post that gives an introduction to transforms, loaders and all.